Store/src/findtarget/findtarget.cs

```diff
@@ -1,68 +1,64 @@
 using CounterStrikeSharp.API.Core;
 using CounterStrikeSharp.API.Modules.Commands;
 using CounterStrikeSharp.API.Modules.Commands.Targeting;
-using static Store.Store;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace Store;
-
-public static class FindTarget
+namespace Store

```

Comment:
If we are going to decorate the Namespace command this way, we need to set them all this way. I don't think it is something very important.